IdentityCore/src/requests/sdk/MSIDBrokerResponseHandler.m

```diff
@@ -309,4 +311,16 @@ - (BOOL)canHandleBrokerResponse:(__unused NSURL *)response
     return YES;
 }
 
+- (MSIDAuthenticationScheme *)synchronizeAuthSchemeFromBrokerResponse:(MSIDAuthenticationScheme *)authSchemeFromResumeState                                                                brokerResponse:(MSIDBrokerResponse *)brokerResponse
+{
+    if (![authSchemeFromResumeState isMemberOfClass:MSIDAuthenticationSchemePop.class]) return authSchemeFromResumeState;
+    
+    NSString *tokenType = [brokerResponse.formDictionary msidObjectForKey:MSID_OAUTH2_TOKEN_TYPE ofClass:[NSString class]];
+    if (!tokenType || MSIDAuthSchemeTypeFromString(tokenType) != MSIDAuthSchemePop)
```

Comment:
Please use the NSString extension method ([NSString msidIsStringNilOrBlank:token_type])